### PR TITLE
Basic Restructuration

### PR DESCRIPTION
**Pull request by AntoineCAU**
Basic Restructuration !

### CODE REVIEW

**Review by Erriep — DISMISSED** [review]
Good job !

**Review by Erriep — DISMISSED** [review]

**Review by Erriep — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint --ext .js,.jsx --fix src/",
+    "dev": "nodemon server.js",
```
Comment (LoicBrassart) [inline package.json]: Not in the frontend, nope

File: package.json [hunk 2]
```diff
@@ -52,6 +53,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "husky": "4.3.8",
     "lint-staged": "^10.5.4",
+    "nodemon": "^2.0.7",
```
Comment (LoicBrassart) [inline package.json]: Not in the frontend, nope

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Stop trying to use `nodemon` in your frontend app

**Review by Erriep — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
NmE!

**Review by Erriep — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE!

**Pull request merged**